src/osmlr.cpp

```diff
@@ -178,21 +189,23 @@ struct tile_exists_filter {
 
 bool check_access(vb::GraphReader &reader, const vb::merge::path &p) {
   // TODO: make traffic mask configurable
+  int i = 0;
```

Comment:
Was this left-over from debugging? It's incremented and used in a `std::cout` print below. Should these be `LOG_INFO` instead, or just removed?